packages/medusa/src/repositories/product-type.ts

```diff
@@ -22,8 +24,35 @@ export class ProductTypeRepository extends Repository<ProductType> {
     const created = this.create({
       value: type.value,
     })
-    const result = await this.save(created)
+    return await this.save(created)
+  }
+
+  async findAndCountByConditionId(
```

Comment:
Method name here could be more explicit that it is a Discount condition `findAndCountByDiscountConditionId`